install-kubevirt-release.yml

```diff
@@ -46,16 +60,19 @@
           --insecure-skip-tls-verify=true \
           "https://${master_fqdn}:8443"
 
-    - name: Install Kubevirt
-      debug:
-        msg: "Installing Kubevirt"
+    - debug:
+        msg: Installing Kubevirt and Kubevirt's testing resources
 
   roles:
-    - "{{ playbook_dir }}/openshift/roles/kubevirt"
+    - role: "{{ playbook_dir }}/openshift/roles/kubevirt"
+    - role: "{{ playbook_dir }}/openshift/roles/kubevirt_testing"
```

Comment:
I think this step should be configurable, as a user I don't want to deploy any resources which are being used for testing.

Reply:
+1